docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -67,6 +65,8 @@ NOTE: See link:usermanaged{outfilesuffix}[the user managed cache documentation]
 
 === Tiering
 
+Ehcache 3, as in previous version, offers a tiering model to allow to store growing amount of data on slower tiers.
```

Comment:
Ehcache 3, as in previous versions, offers a tiering model to allow storing increasing amounts of data on slower tiers.